target/linux/ramips/mt76x8/base-files/etc/board.d/02_network

```diff
@@ -122,9 +122,6 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0:lan:2" "1:lan:1" "4:wan" "6@eth0"
 		;;
-	*)
```

Comment:
This will result in the device becoming a one-port eventually (but a broken one), as there is a fallback to `ucidef_set_interface_lan eth0` hidden somewhere I don't remember (base-files package or something like that).

Rather than having this strange fallback nobody really knows about, I'd prefer to have the one-port setup as default case for the subtargets touched here.

Reply:
For `ucidef_set_interfaces_lan_wan "eth0.1" "eth0.2"` case, the VLAN will be set according to the dts AFAIK, so the effect will be same as `ucidef_set_interfaces_lan_wan "lan1 lan2 lan3 lan4" "wan"` in the default case of mt7621 02_network. (But it requires the preinit script, so it's broken now.)

> Rather than having this strange fallback nobody really knows about, I'd prefer to have the one-port setup as default case for the subtargets touched here.

Do you mean setting `ucidef_set_interface_lan eth0` explicitly as a default case?

Reply:
Instead of just not having any default case here, I'd choose the
```
		ucidef_add_switch "switch0"
		ucidef_add_switch_attr "switch0" "enable" "false"
		ucidef_set_interface_lan "eth0"
		;;
```

Reply:
Okay, understood.